calico/observability/view-flow-logs.mdx

```diff
@@ -3,4 +3,98 @@ description: View flow logs in the Calico Whisker web console.
+While the console is open, it receives live data from a dedicated flow logs API (Goldmane).
+You can filter the stream to see only the logs for a particular policy, namespace, pod, or other criteria.
+
+<Screenshot src="/img/calico/whisker-flows.png" alt="A view of the Calico Whisker web console" />
+*Figure 1. A view of the Calico Whisker web console showing flow logs in a Kubernetes cluster.*
+
+### Flow log data types
+
+The following table describes the flow log data types used by Calico Whisker.
+
+| Name                              | Datatype          | Description                                                                                                                                                                                                                                                                                                                                          |
+| --------------------------------- | ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `start_time`                      | date              | Start time of log collection in UNIX timestamp format.                                                                                                                                                                                                                                                                                               |
+| `end_time`                        | date              | End time of log collection in UNIX timestamp format.                                                                                                                                                                                                                                                                                                 |
+| `action`                          | keyword           | - `allow`: Calico accepted the flow.<br />- `deny`: Calico denied the flow.                                                                                                                                                                                                                                                              |
+| `bytes_in`                        | long              | Number of incoming bytes since the last export.                                                                                                                                                                                                                                                                                                      |
```

Comment:
```suggestion
| `bytes_in`                        | long              | Number of incoming bytes within the timeframe.                                                                                                                                                                                                                                                                                                   |
```

It might be equivalent? But this is more accurate - i.e., what is "since the last export" for the first flow log you receive?

Reply:
Ok. I've modified the line. This and all the other data types are lifted from the CE docs. It's good to adjust where necessary.

Reply:
Changed all four instances.